services/__tests__/TimeEntriesApi.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { getTimeEntries } from "../time-entries-api";
+
+const mockedTimeEntries = [
```

Comment:
Move mocks to a __mock__ directory

https://jestjs.io/docs/manual-mocks
https://stackoverflow.com/questions/51303189/how-to-move-mocks-folder-in-jest-to-test